docs/self-managed/modeler/web-modeler/configuration/configuration.md

```diff
@@ -44,6 +44,18 @@ Web Modeler requires an SMTP server to send notification emails to users.
 | `RESTAPI_MAIL_FROM_NAME`    | [optional]<br/>Name displayed as the sender of emails sent by Web Modeler.                             | `Camunda`                     | `Camunda`     |
 | `RESTAPI_SERVER_URL`        | URL at which users access Web Modeler in the browser (used to construct links in notification emails). | `https://modeler.example.com` | -             |
 
+#### Collaboration and invitation limits
```

Comment:
I feel like this is the only point in the SM docs we mention some form of collaboration, so I've placed this content here for now.

Alternatively, if we want a SM-specific doc outlining collaboration like we do [here](https://docs.camunda.io/docs/components/modeler/web-modeler/collaboration/), we should likely create a separate issue to document this.

Reply:
We already shipped with `alpha2` a link to the documentation which points to the collaboration page. And I think the correct place is the collaborator page as we discussed this internally already.

CC: @wollefitz

Reply:
If there has already been internal discussion on this I'm fine with keeping it as-is, but worry as this is specific to Self-Managed.

Reply:
In my opinion, it doesn't make sense to mention the limits on the configuration page as it should be focused on configuration properties only. The restriction don't necessarily have to be mentioned on a "Self-Managed" collaboration page - this is more about **licensing** than it is about **collaboration**.

@JacobiCamunda What should still be mentioned is the new environment variable `CAMUNDA_LICENSE_KEY` consumed by both `webapp` and `restapi` with a link to the page explaining restrictions.

> If there has already been internal discussion on this I'm fine with keeping it as-is, but worry as this is specific to Self-Managed.

What going forward:
* we put the note about collaboration limits on the page which is referenced in the Web Modeler `8.6-alpha2` updates (ie. `docs/components/modeler/web-modeler/collaboration.md`) for the `8.6-alpha2` release
* we follow up on this topic immediately afterwards in scope of https://github.com/camunda/product-hub/issues/2286

Reply:
I'll move the content over to the collaboration doc 👍